lib/Runtime/Library/JSONStringifier.cpp

```diff
@@ -520,18 +552,25 @@ JSONStringifier::ReadObject(_In_ RecyclableObject* obj, _In_ JSONObjectStack* ob
             JavascriptStaticEnumerator enumerator;
             if (obj->GetEnumerator(&enumerator, EnumeratorFlags::SnapShotSemantics | EnumeratorFlags::EphemeralReference, this->scriptContext))
             {
-                enumerator.GetInitialPropertyCount();
```

Comment:
You might want to clean up the implementation in JAvascriptStaticEnumerator.h and DynamicObjectPropertyEnumerator.h as well, as I don't see other references to those.